Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -128,5 +150,8 @@ void vtkMRMLClipModelsNode::PrintSelf(ostream& os, vtkIndent indent)
   os << indent << "YellowSliceClipState: " << this->YellowSliceClipState << "\n";
   os << indent << "GreenSliceClipState:  " << this->GreenSliceClipState << "\n";
   os << indent << "RedSliceClipState:    " << this->RedSliceClipState << "\n";
+  os << indent << " clippingMethod=\"" << ((this->ClippingMethod == vtkMRMLClipModelsNode::
+    Straight) ? "Straight" : ((this->ClippingMethod == vtkMRMLClipModelsNode::
+    WholeCells) ? "WholeCells" : "WholeCellsWithBoundary")) << "\n";
```

Comment:
Yes, separate methods are needed for enum<->string conversion, see my comments above.